crowbar_framework/spec/models/api/upgrade_spec.rb

```diff
@@ -53,25 +54,25 @@
   context "with a successful status" do
     it "checks the status" do
       allow(Crowbar::Sanity).to receive(:sane?).and_return(true)
-      allow_any_instance_of(Api::Crowbar).to(
+      allow(Api::Crowbar).to(
         receive(:features).and_return([])
       )
 
+      allow(Api::Crowbar).to(
+        receive(:addons).and_return(
+          ["ha"]
+        )
+      )
+      allow(Api::Crowbar).to(
+        receive(:ha_presence_check).and_return({})
+      )
+
       expect(subject.class).to respond_to(:status)
       expect(subject.class.status).to be_a(Hash)
       expect(subject.class.status.deep_stringify_keys).to eq(upgrade_status)
     end
   end
 
-  context "with a successful maintenance updates check" do
```

Comment:
In the backport PR (https://github.com/crowbar/crowbar-core/pull/689) it is in the separate commit. I thought you wanted this PR to be gone (thus I created https://github.com/crowbar/crowbar-ha/pull/147 for master.